types/api/v1beta1/event_data.proto

```diff
@@ -0,0 +1,99 @@
+syntax = "proto3";
+
+option go_package = "github.co/aquasecurity/tracee/api/v1beta1";
+
+package tracee.v1beta1;
+
+import "google/protobuf/wrappers.proto";
+
+message EventValue {
+    oneof value {
```

Comment:
I'm missing types for triggerEvent and for args (array of ordered system call arguments)